types/gestalt/index.d.ts

```diff
@@ -1333,7 +1338,7 @@ export interface TextAreaProps {
     id: string;
     onChange: (args: { event: React.SyntheticEvent<HTMLTextAreaElement>; value: string }) => void;
     disabled?: boolean;
-    errorMessage?: string;
+    errorMessage?: React.ReactNode | string;
```

Comment:
nit: `React.ReactNode` includes `string` type so `React.ReactNode | string` is redundant.